docs_source_files/content/grid/grid_4/advanced_features/observability.en.md

```diff
@@ -0,0 +1,147 @@
+ * Timestamp
+ * Logging level
+ * Logger class
+ * Log message (This is further broken down into fields relevant to the operation where the log was recorded)
+
+Logs and events are closely related. Events encapsulate all the possible information available to do a single unit of work. Logs are essentially subsets of an event. At the crux, both aid in debugging.
+Refer following resources for detailed understanding:
+ 1. [https://www.honeycomb.io/blog/how-are-structured-logs-different-from-events/](https://www.honeycomb.io/blog/how-are-structured-logs-different-from-events/)
+ 2. [https://charity.wtf/2019/02/05/logs-vs-structured-events/](https://charity.wtf/2019/02/05/logs-vs-structured-events/)
+
+## Grid Observability
+
+Selenium server is instrumented with tracing using OpenTelemetry. Every request to the server is traced from start to end. Each trace consists of a series of spans as a request is executed within the server. 
+Most spans in the Selenium server consist of two events:
+1. Normal event - records all information about a unit of work and marks successful completion of the work.
+2. Error event - records all information till the error occurs and then records the error information. Marks an exception event.
+
+Running Selenium server 
+ 1. [Standalone](https://github.com/SeleniumHQ/selenium/wiki/Selenium-Grid-4#standalone-mode)
+ 2. [Hub and Node](https://github.com/SeleniumHQ/selenium/wiki/Selenium-Grid-4#hub-and-node)
+ 3. [Fully Distributed](https://github.com/SeleniumHQ/selenium/wiki/Selenium-Grid-4#fully-distributed)
+ 4. [Docker](https://github.com/SeleniumHQ/selenium/wiki/Selenium-Grid-4#using-docker)
+
+## Visualizing Traces
+All spans, events and their respective attributes are part of a trace. Tracing works while running the server in all of the above-mentioned modes.
+
+By default, tracing is enabled in the Selenium server. Selenium server exports the traces via two exporters:
+1. Console - Logs all traces and their included spans at FINE level. By default, Selenium server prints logs at INFO level and above. As per need, this can be changed via [LoggingOptions.java](https://github.com/SeleniumHQ/selenium/blob/738dd0ee18a142d3fe69fc5f16889ac770188c93/java/server/src/org/openqa/selenium/grid/log/LoggingOptions.java). Add a line `logger.setLevel(Level.FINE);` Pick a logging level of choice.
+2. Jaeger UI - OpenTelemetry provides the APIs and SDKs to instrument traces in the code. Whereas Jaeger is a tracing backend, that aids in collecting the tracing telemetry data and providing querying, filtering and visualizing features for the data.
+
+[Detailed instructions of visualizing traces using Jaeger UI](https://github.com/SeleniumHQ/selenium/blob/a28b4beeeed965a352025a09f26fe2a9567cdeb3/java/server/src/org/openqa/selenium/grid/commands/tracing.txt)
```

Comment:
Maybe reference the `selenium info tracing` command? That'll be up-to-date with the version of selenium that someone is using.